src/platform_impl/macos/window.rs

```diff
@@ -1241,6 +1255,20 @@ impl WindowExtMacOS for UnownedWindow {
                 .setHasShadow_(if has_shadow { YES } else { NO })
         }
     }
+
+    #[inline]
+    fn set_background_color(&self, red: u8, green: u8, blue: u8, alpha: u8) {
+        autoreleasepool(|| unsafe {
+            let ns_background = NSColor::colorWithSRGBRed_green_blue_alpha_(
+                nil,
+                red as f64 / 255f64,
+                green as f64 / 255f64,
+                blue as f64 / 255f64,
+                alpha as f64 / 255f64,
+            );
+            self.ns_window.setBackgroundColor_(ns_background);
```

Comment:
This is very likely not thread safe, and should always be done on the main thread. As an example, try search for `set_title_async` to see how we do that.